app/views/components/forms/edit/_builder.html.erb

```diff
@@ -216,6 +216,17 @@ $(function() {
     });
   });
 
+  $(".form-delete-question-option").on("click", function(event) {
+    event.preventDefault();
+    if (confirm('Are you sure?')) {
+      $(this).parent().parent().remove();
```

Comment:
Is there another way to target the element, rather than `.parent().parent()`?

Reply:
closest.find